### PR TITLE
gitea auto app.ini + admin user

### PR DESCRIPTION
**Pull request by henderiw**
This is a more complete installation of Gitea which auto configures app.ini and initialises a default admin user.
It is aligned with the hem charts which use memcached/postgres.

### CODE REVIEW

File: gitea/secret-gitea-init.yaml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        echo 'no ldap configuration... skipping.'
+    }
+
+    configure_ldap
+
+    function configure_oauth() {
+        echo 'no oauth configuration... skipping.'
+    }
+
+    configure_oauth
```
Comment (electrocucaracha) [inline gitea/secret-gitea-init.yaml:70]: These functions are doing nothing, we can keep it for helm compatibility or remove them to simplify the script.

Comment (henderiw) [reply gitea/secret-gitea-init.yaml:70]: I prefer to keep it since the helm charts has them for now. But agreed they are not used for us.

**Comment by johnbelamaric** [comment]
Merged and tagged as gitea/v2

**Pull request merged**